core/src/services/mysql/backend.rs

```diff
@@ -245,8 +263,8 @@ impl kv::Adapter for Adapter {
 
     async fn set(&self, path: &str, value: &[u8]) -> Result<()> {
         let query = format!(
-            "INSERT INTO `{}` (`{}`, `{}`) 
-            VALUES (:path, :value) 
+            "INSERT INTO `{}` (`{}`, `{}`)
```

Comment:
Please don't change unrelated part of code.

Reply:
My editor will auto delete  the whitespace at end of lines.Could I close this feature? 

Then add orig whitespace to these lines?

Reply:
Not a big issue, let's ignore it.